docs2/site/docs/getting-started/errors.md

```diff
@@ -165,54 +165,10 @@ options.UnhandledExecutionDelegate = ctx =>
-object tree into json. The GraphQL spec allows for four properties to be returned within each
-error: `message`, `locations`, `path`, and `extensions`. The `IDocumentWriter` implementations
-provided for the [`Newtonsoft.Json`](https://www.nuget.org/packages/GraphQL.NewtonsoftJson) and
-[`System.Text.Json`](https://www.nuget.org/packages/GraphQL.SystemTextJson) packages allow you to control the
-serialization of `ExecutionError`s into the resulting json data by providing an `IErrorInfoProvider`
-to the constructor of the document writer. The `ErrorInfoProvider` class (default implementation of
-`IErrorInfoProvider`) contains 5 properties to control serialization behavior:
-
-* `ExposeExceptionStackTrace` when enabled sets the `message` property for errors to equal the
-exception's `.ToString()` method, which includes a stack trace. This property defaults to `false`.
-* `ExposeCode` when enabled sets the `extensions`'s `code` property to equal the error's `Code`
-property. This property defaults to `true`.
-* `ExposeCodes` when enabled sets the `extensions`'s `codes` property to equal a list containing both
-the error's `Code` property, if any, and the type name of inner exceptions (after being converted to
-UPPER_CASE and removing the "Extension" suffix). So an `ExecutionError` with a code of `INVALID_FORMAT`
-that has an inner exception of type `ArgumentNullException` would contain a `codes` property
-of `["INVALID_FORMAT", "ARGUMENT_NULL"]`. This property defaults to `true`.
-* `ExposeData` when enabled sets the `extension`'s `data` property to equal the data within the error's
-`Data` property. This property defaults to `true`.
-* `ExposeExtensions` when disabled hides the entire `extensions` property, including `code`, `codes`,
-and `data` (if enabled). This property defaults to `true`.
-
-For example, to show the stack traces for unhandled errors during development, you might write code like this:
-
-```csharp
-#if DEBUG
-    var documentWriter = new DocumentWriter(true, new ErrorInfoProvider(options => options.ExposeExceptionStackTrace = true));
-#else
-    var documentWriter = new DocumentWriter();
-#endif
-```
-
-You can also write your own implementation of `IErrorInfoProvider`. For instance, you might want to override
-the numerical codes provided by GraphQL.NET for validation errors, reveal stack traces
-only to logged-in administrators, or simply add information to the returned error object. Below is a sample
-of a custom `IErrorInfoProvider` that adds a date stamp to returned error objects:
-
-```csharp
-public class MyErrorInfoProvider : GraphQL.Execution.ErrorInfoProvider
-{
-    public override ErrorInfo GetInfo(ExecutionError executionError)
-    {
-        var info = base.GetInfo(executionError);
-        info.Extensions["timestamp"] = DateTime.Now.ToString("u");
-        return info;
-    }
-}
-```
+object tree into json. The `IDocumentWriter` implementations provided by `GraphQL.SystemTextJson`
```

Comment:
>  The `IDocumentWriter` implementations provided by `GraphQL.SystemTextJson`
and `GraphQL.NewtonsoftJson` allow for configuration of error serialization by providing an
`IErrorInfoProvider` implementation.

IMO something wrong in wording in this sentence.

Reply:
Only this one left. I'm not sure I read this sentence correctly. Everything else is fine.
Maybe that way?

The IDocumentWriter implementations provided by both GraphQL.SystemTextJson
and GraphQL.NewtonsoftJson packages allow you to configure error serialization by taking an IErrorInfoProvider dependency.

Reply:
I made some more changes; please take a look.